benchmarks/benchmarks.c

```diff
@@ -55,3 +54,188 @@ void benchmarkFillup ()
 		}
 	}
 }
+
+#ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
+
+// needed for the opmphmRandom ()
+#include "../src/libs/elektra/opmphm.c"
+
+/**
+ * @brief Fills a string up with random chars and null terminates it.
+ *
+ * @param name the string
+ * @param length the length of the string (excluding the '\0')
+ * @param seed to generate random data
+ *
+ */
+static void fillUpWithRandomChars (char * name, int length, uint32_t * seed)
+{
+	int i = 0;
+	char * c = name;
+	while (i < length)
+	{
+
+		if ((opmphmRandom (seed) % 10) == 0)
```

Comment:
This breaks uniformity of the random numbers. Maybe we can change the signature so that we can tell between which numbers we want a number?

We should also reconsider it to be more than 32bit, because this limits the maximum length of a random number sequence.